test/validationTest.lua

```diff
@@ -122,129 +122,159 @@ local json_capabilities = {
--- --   --validator.validate_mobile_response( "WrongFunctionName", { success = false, resultCode = "INVALID_DATA", info = nil })
--- -- if _res then
--- --   print("validate_mobile_response with \"WrongFunctionName\":"..tostring(_res))
--- -- else
--- --   print("validate_mobile_response with \"WrongFunctionName\":"..tostring(_res).." ==> ".._err)
--- -- end
-
--- _res, _err = hmi_schema:Validate("BasicCommunication.OnSystemRequest", "HMI", 'notification',  { requestType = "PROPRIETARY"}) -- validator.validate_hmi_notification( 'BasicCommunication.OnSystemRequest', { requestType = {"PROPRIETARY"}})
--- if _res then
---   print("validate_hmi_notification:"..tostring(_res))
--- else
---   print("validate_hmi_notification:"..tostring(_res).." ==> ".._err)
--- end
-
--- _res, _err = hmi_schema:Validate('BasicCommunication.OnSystemRequest', "HMI", "notification", { urlSchema = "default", fileName = "fileName"}, true)
---  -- validator.validate_hmi_notification( 'BasicCommunication.OnSystemRequest', { urlSchema = "default", fileName = "fileName"}, true)
-
--- if _res then
---   print("validate_hmi_notification:"..tostring(_res))
--- else
---   print("validate_hmi_notification:"..tostring(_res).." ==> ".._err)
--- end
-
--- _res, _err =mob_schema:Validate("OnHMIStatus", "MOBILE", "notification", {hmiLevel = "FULL"})
---  -- validator.validate_mobile_notification( "OnHMIStatus", {hmiLevel = "FULL"})
--- if _res then
---   print("validate_modile_notification:"..tostring(_res))
--- else
---   print("validate_mobile_notification:"..tostring(_res).." ==> ".._err)
--- end
-
--- _res, _err = validator.json_validate(json_mob_tbl, {resultCode = {"SUCCESS"} })
--- if _res then
---   print("json_validate:"..tostring(_res))
--- else
---   print("json_validate:"..tostring(_res).." ==> ".._err)
--- end
-
--- _res, _err = validator.json_validate(json_mob_tbl, { success = 'true', resultCode = {"SUCCESS"} } )
--- if _res then
---   print("json_validate:"..tostring(_res))
--- else
---   print("json_validate:"..tostring(_res).." ==> ".._err)
--- end
-
--- local _res, _err = validator.validate_mobile_request('PerformInteraction',
---   { initialText = "initialText",
---     initialPrompt = {{text = "initialPrompt", type = "TEXT"}},
---     interactionMode = "MANUAL_ONLY",
---     interactionChoiceSetIDList = {1},
---     helpPrompt = {{text = "helpPrompt", type = "TEXT"}},
---     interactionLayout = "KEYBOARD"
---   })
--- if _res then
---   print("validate_hmi_request:"..tostring(_res))
--- else
---   print("validate_hmi_request:"..tostring(_res).." ==> ".._err)
--- end
+print("======================= Negative case \n")
+_res, _err = mob_schema:Validate('Slider', "MOBILE", 'response', json_mob_tbl, true)
+if _res then
+  print("validate_mobile_response:"..tostring(_res))
+else
+  print("validate_mobile_response:"..tostring(_res).." ==> ".._err)
+end
 
```

Comment:
@VVeremjova Please add some complicated tests like array of structs, structs with arrays.